src/App.css

```diff
@@ -62,17 +62,24 @@ html, body {
 @keyframes animate {
 
     0%{
```

Comment:
Can we move everything outside of App.css? I tried deleting this file before, but it seemed to mess up the text weights so I left it in. Ideally we should only have either styled components or .css files taking care of the styles, not both.

Reply:
I moved the animation as a styled component itself. That makes sense, thanks!